spec/index.adoc

```diff
@@ -2896,8 +2896,8 @@ else
   S with
     messages = Older_messages · Younger_messages
     balances[M.receiver] =
-      (S.balances[M.receiver] + (if Is_response then MAX_CYCLES_PER_RESPONSE else MAX_CYCLES_PER_MESSAGE))
-      - Cycles_used
+      S.balances[M.receiver] +
+      ((if Is_response then MAX_CYCLES_PER_RESPONSE else MAX_CYCLES_PER_MESSAGE) - Cycles_used)
```

Comment:
@bogwar Could you please confirm that this corresponds to what the mainnet implementation does?

Reply:
Thank you for the confirmation!